autogen/agentchat/contrib/agent_eval/agent_eval.py

```diff
@@ -0,0 +1,101 @@
+from typing import Dict, List, Optional, Union
+
+import autogen
+from autogen.agentchat.contrib.agent_eval.criterion import Criterion
+from autogen.agentchat.contrib.agent_eval.critic_agent import CriticAgent
+from autogen.agentchat.contrib.agent_eval.quantifier_agent import QuantifierAgent
+from autogen.agentchat.contrib.agent_eval.subcritic_agent import SubCriticAgent
+from autogen.agentchat.contrib.agent_eval.task import Task
+
+
+def generate_criteria(
+    llm_config: Optional[Union[Dict, bool]] = None,
```

Comment:
`llm_config`'s type hint for conversable agents is `Optional[Union[Dict, Literal[False]]]`